assets/sass/components/_inset-text.scss

```diff
@@ -0,0 +1,4 @@
+.govuk-inset-text--grey {
```

Comment:
The [suggested naming for custom modifiers to GOV.UK Frontend components](https://design-system.service.gov.uk/get-started/extending-and-modifying-components/#small-modifications-to-components) I think would make this `.app-inset-text--grey`. I might have misunderstood the guidance though; does your reading of it match?

Reply:
I didn't know this, but I think you're right. Have changed this.